code/__DEFINES/maths.dm

```diff
@@ -34,6 +34,8 @@
 /// Gets the sign of x, returns -1 if negative, 0 if 0, 1 if positive
 #define SIGN(x) ( ((x) > 0) - ((x) < 0) )
 
+#define CEIL(x) (-round(-(x)))
```

Comment:
couldn't you use CEILING(x, 1) for this?

Reply:
maybe

Reply:
actually ROUND_UP seems to be an identical macro so we'll just use that and rid of this